pkg/util/common.go

```diff
@@ -30,12 +30,107 @@ import (
 
 	"github.com/blang/semver"
 	"github.com/sirupsen/logrus"
+
+	"k8s.io/release/pkg/command"
 )
 
 const (
 	TagPrefix = "v"
 )
 
+// PackagesAvailable takes a slice of packages and determines if they are installed
+// on the host OS. Replaces common::check_packages.
+func PackagesAvailable(packages ...string) (bool, error) {
+	hostOS, osErr := getOS()
+	if osErr != nil {
+		return false, osErr
+	}
+
+	var pkgMgr string
+	missingPkgs := []string{}
+
+	ok := true
+	switch hostOS {
```

Comment:
We could add a `default` block in this `switch` statement to go ahead and return `false` and an error if we are only supporting these OS'es below. If I read this function correctly, it looks like if the OS is not one of these four, then we will always return `true, nil` because `len(missingPkgs)` will be zero.

Reply:
@hasheddan -- Great catch! Fixed in 594773f6c6e3af02f2ea1aa1ed18465dd2a2751d.